tests/integration/devfile/cmd_devfile_registry_test.go

```diff
@@ -80,6 +81,27 @@ var _ = Describe("odo devfile registry command tests", func() {
 			helper.CmdShouldPass("odo", "registry", "delete", registryName, "-f")
 			helper.CmdShouldFail("odo", "create", "java-maven", "--registry", registryName)
 		})
+	})
 
+	Context("when working with git based registries", func() {
+		It("should show deprecation warning when when the git based registry is used", func() {
```

Comment:
shouldn't we clean up the registris that were added in these tests?

Reply:
Registries are stored in the preference file and a new preference is created for each test. So I don't think this would be a problem

https://github.com/openshift/odo/blob/796bd2da69904f722b73b4c559b2cc96e18b3318/tests/helper/helper_generic.go#L300-L303

Reply:
Anything else @girishramnani ?